src/sentry/api/endpoints/group_details.py

```diff
@@ -79,7 +79,7 @@ def _get_activity(self, request, group, num):
         ).order_by('-datetime').select_related('user')
         # we select excess so we can filter dupes
         for item in activity_qs[:num * 2]:
-            sig = (item.type, item.ident, item.user_id)
+            sig = (item.type, item.ident, item.user_id, hash(frozenset(item.data.items())))
```

Comment:
@MaxBittker What's going on here.

Reply:
it needs to use data or it filters things as dupes incorrectly